api/write-raw.md

```diff
@@ -0,0 +1,116 @@
+# Writing Raw Data
+
+In contrast to the other submission APIs ([numeric](./write-nnt.md),
+[text](./write-text.md), [histogram](./write-histogram.md)), which accept
+pre-rolled data, the raw API accepts direct input of metric records at
+arbitrary frequencies. It stores every metric data point as it was received,
+for a configurable amount of time, before aging it out to a rollup format.
+
+Metric records are in one of several formats, and are accepted as either
+tab-separated values or as FlatBuffer messages.
+
+## Metric Record Formats
+Raw metric records may be submitted in one of several formats, depending on the
+type of metric data contained within.
+
+### TSV Format
+
+**URI:** /raw
+
+**Method:** PUT | POST
+
+**Format:** Multiple (see below)
+
+#### TSV Numeric Or Text Metrics
+
+Individual numeric or text metrics submitted to the raw endpoint as lines of
+ASCII characters use the following format, referred to as an `M` record:
+```
+M TIMESTAMP UUID NAME TYPE VALUE
+```
+
+Components are separated by TAB characters. Multiple records may be sent in the
+same operation, separated by newline (`\n`).
+
+*M* : Denotes an `M` record.
+
+*TIMESTAMP* : An epoch timestamp recording the time of the observation, with
+milliseconds. In terms of format, it is `%lu.%03lu`, i.e., `1516820826.120`.
```

Comment:
I would further note that this might look like a number, but in fact it is a strict texutal format requiring exactly three digits past the decimal point.  They must always been included even if they are 000

Reply:
Also worth pointing out numeric data which collides on NAME/TIMESTAMP (same name and millisecond) will keep the larger by absolute value in the default configuration.